### ISSUE TITLE
[Adaptive Card iOS - ColumnSet.Input.Toggle.VerticalStretch.json]: Label text 'Placeholder' for toggle button is not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7347]
https://microsoft.visualstudio.com/OS/_workitems/edit/37842426
